slack_notifier.go

```diff
@@ -116,6 +116,9 @@ func SlackNotifierEntryPoint(ctx context.Context, m *pubsub.Message) error {
 	})
 
 	for url := range urls {
+		if len(url) == 0 {
```

Comment:
This actually does not suppress alerts you want to suppress.
For example, rule 2 below will be skipped, but the exact same name will be captured by rule 1.

Rule1:

regex: .*
url: valid webhook URL
Rule2:
regex: prefix-you-want-to-suppress-.+
url: empty